src/common/components/textInput/InputWrapper.tsx

```diff
@@ -76,7 +76,11 @@ const InputWrapper: FC<InputWrapperProps> = React.forwardRef<
         </Tooltip>
       )}
       <div className={classNames(styles.inputWrapper)}>{children}</div>
-      {helperText && <div className={styles.helperText}>{helperText}</div>}
+      {helperText && (
+        <div className={styles.helperText} role="alert">
```

Comment:
`role="alert"` added for accessibilty + easier testing

Reply:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_alert_role

"The alert role is used to communicate an important and usually time-sensitive message to the user."

The problem here is that this helperText element could appear to many input at the same time, I'm not sure if that is  good user experience with screen readers. Also helperText element is used for other than errors so we would have elements with role="alert" that are not really alert messages